### DISCUSSION TITLE
I'm really lost help needed about new Update :)

### DISCUSSION THREAD

**Discussion by EKKIV0K** [vladmandic/sdnext#2808] [Q&A]
hello everyone and thanks in advance for your answers !
i always used the old version (old school boy) my last version (old of 5 month) where lost when i formated my computer.
i tried to reinstall it but so many problem of git and model cannot be loaded occured (i dunno why) 
cause it's worked like a charm before the reinstallation of windows.

Here is my problems that i can't figure out, 

1st - is there a trick to get the old UI look ? 
The new one is a bit weird and preview and generated images window is smaller than the old one .....

2nd - where is my lycroris models ?! inb the old one there was a category Lycoris and now it doesn't show up.....
(even if i install the outdated sd-webui-lycoris extension) no lycroris tab at all .....

3rd - what is Brownian & discard penultimate sigma ????
in the old one there was nothing like this, now i need to be a kind of einstein to use it ....xD 
why the simple samplers disapeared ?! now i need to try every option to get the best images !
to much math here, inb the old one there DPM++2M and karras and that's all for me and it's working great ! 

I'm so lost right now, and it's frustrating to have no possibilities to go back to an older version before that....
i tried so many things but there is always errors with git or modules stuffs .....it's working great 2 weeks ago....

thanks guys for your help 👍

**Comment by vladmandic** [comment]
- there are tons of themes, pick one that you like.
- there is no lycoris tab and lycoris extension is not supported because support is built-in. there are 10+ lora variations by now, i don't want to have a tab for each variation. lora, lokr, loha, lyco, iah3, etc....
- "simple dpm++ 2m karras" well, that's the problem - karras is not a scheduler, it never was. there is a short list of actual schedulers and then some of them have additional options. old way is to combine matrix of all options and present that as a flat list - it doesn't work when that list would now be 50+ entries. so you want dpm++2m with karras. select that.

**Comment by EKKIV0K** [comment]
@vladmandic 

- Ok but what is Brownian & discard penultimate sigma ?
- and how to get a more larger generation window ? because it turn into smallllllll preview window ..
- Is there a way to reproduce old seeds ?

**Comment by vladmandic** [comment]
- brownian: should initial noise be created using brownian distribution. and for more details, lookup what is brownian noise on wikipedia
- disard penultimage: since no scheduler is ever perfect, should we discard remaining noise at last step
- small preview - its increased in dev, will be part of the next release
- reproduce old seeds? sure. but it totally depends on your old params how to map them to app that is now half a year newer with tons of modifications.

**Reply by EKKIV0K** [reply]
@vladmandic hummm i see, 

i checked the definition on wikipedia but it's like nuclear physics here.....
can you just explain me what is on the purpose of stable diffusion ? 
is it better to use it or not ? same for the other one the......hmmmm...penultimage.....(omg the name of that stuff) ...

**Reply by vladmandic** [reply]
its a personal preference. if there was a single-best that is clearly superior, i wouldn't expose options.
